storage/wdpost_run.go

```diff
@@ -431,8 +431,7 @@ func (s *WindowPoStScheduler) runPost(ctx context.Context, di miner.DeadlineInfo
 		return nil, err
 	}
 
-	// TODO: Faults!
```

Comment:
Confirm you meant to take this TODO out?

Reply:
Yes